### DISCUSSION TITLE
Integrate "Symflower" for Go and Java test generation

### DISCUSSION THREAD

**Discussion by zimmski** [unit-mesh/auto-dev#159] [Ideas]
Currently working on combining LLM chains with Symflower e.g. https://twitter.com/zimmskal/status/1782409124757520527 Maybe this project could do an integration as well so we can get more uses cases to work? Idea is to use `symflower unit-test-skeletons` on specific method, and use that template (which uses the correct test-framework, types, style, ...) for generating more tests.

**Comment by phodal** [comment]
Welcome to PR~~

I had did some research in  [https://github.com/JetBrains-Research/TestSpark](https://github.com/JetBrains-Research/TestSpark) with  EvoSuite before, but it only support gen Java test gen. 

So if you would like to give some PR would be helpful.
 
But actually, AutoDev will analysis dependencies and spike which it's template for current test, like which is for controller, service and repository.
